examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/EventHubService.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Azure.Identity;
+using Azure.Messaging.EventHubs;
+using Azure.Messaging.EventHubs.Consumer;
+using Azure.Messaging.EventHubs.Processor;
+using Azure.Storage.Blobs;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using Microsoft.Extensions.Options;
+using System;
+using System.Threading.Tasks;
+
+namespace WebDemoWithEventHub
+{
+    public class EventHubService
+    {
+        private Settings _settings;
```

Comment:
We don't need this.

Reply:
Removed it.

---
In reply to: [427598190](https://github.com/Azure/AppConfiguration/pull/329#discussion_r427598190) [](ancestors = 427598190)